docker-compose.yml

```diff
@@ -1,7 +1,6 @@
 version: "3"
 volumes:
   shuttle-backend-vol:
-    external: true
```

Comment:
Shouid we also remove the `docker volume create shuttle-backend-vol` from `CONTRIBUTING.md`?